.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true && github.event.pull_request.head.ref == 'release'
```

Comment:
Without the if statement, the release job will work for all PRs. Therefore, some condition is required. I have made it a release branch now, but it can be anything else.
Do you have any condition?

Reply:
Can we do comparing version and git tag?
version=get_version()
if a tag exists corresponding to the version then do nothing,
otherwise do the release process.

Reply:
Ok, I added check version.
This compares the version obtained by get_version with the github tag, and if it already exists, stops the workflow there.

https://github.com/line/cosmwasm/pull/108/commits/4f95a4dd00e0361a52e12c85d14f1f1f0ebe97f7#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R34